src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -58,9 +141,30 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
                 Padding = new Thickness(1),
                 VerticalAlignment = VerticalAlignment.Center,
             };
-
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = sender as Border;
+            var uiList = CreateDescriptionAsync().Result;
```

Comment:
This is potentially a performance issue because the .Result can block. I suspect we can do the following:

1. Switch this event handler to be async void, which is normally a no-no but this is an event handler so that's OK.
2. Do the e.Handled = true right away, before an await. That way the WPF knows we're handling it, but we're continuing to process async.